src/formatter/__test__/mergeFields.test.ts

```diff
@@ -0,0 +1,42 @@
+import { mergeFields } from "../mergeFields";
+
+const inputData = {
+  bar: 1,
+  baz: "hello",
+  foo: "Foo"
+};
+
+describe("mergeFields", () => {
+  it("should merge fields into a new field", () => {
+    const res = mergeFields(inputData, ["foo", "bar"], "mergedField", "-");
+    expect(res).toEqual({
+      bar: 1,
+      baz: "hello",
+      foo: "Foo",
+      mergedField: "Foo-1"
+    });
+  });
+  it("should output an empty string if no fields are provided", () => {
+    const res = mergeFields(inputData, [], "mergedField", "-");
+    expect(res).toEqual({
+      bar: 1,
+      baz: "hello",
+      foo: "Foo",
+      mergedField: ""
+    });
+  });
+  it("should output an empty string if a missing field is provided", () => {
```

Comment:
What do you think about this behavior @drmarro @CrisTofani ? Do you prefer to return an empty string or returning the same input with no new field added to data stream?

Reply:
I think that if no edit happens we should know, maybe an empty array would be more representative